gaurav/package.json

```diff
@@ -10,21 +10,28 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@radix-ui/react-slot": "^1.1.0",
     "@reduxjs/toolkit": "^2.3.0",
+    "class-variance-authority": "^0.7.1",
+    "clsx": "^2.1.1",
+    "date-fns": "^3.6.0",
     "lucide-react": "^0.460.0",
     "react": "^18.3.1",
+    "react-day-picker": "^8.10.1",
     "react-dom": "^18.3.1",
-
-    "react-router-dom": "^7.0.1",
-    "serve": "^14.2.4",
-    "vite-plugin-pwa": "^0.21.0",
-
     "react-redux": "^9.1.2",
-    "redux-persist": "^6.0.0"
+    "react-router-dom": "^7.0.2",
+    "redux-persist": "^6.0.0",
+    "serve": "^14.2.4",
```

Comment:
what's the use of `serve`?

Reply:
`serve` is used to setting up local servers to host static web projects that are in development.
Removed as it is remained unused